### PR TITLE
HSEARCH-2759 + HSEARCH-2847 : ITs regarding how GeoPointBridge accesses entity data 

### PR DESCRIPTION
**Pull request by yrodiere**
https://hibernate.atlassian.net/browse/HSEARCH-2759
https://hibernate.atlassian.net/browse/HSEARCH-2847

~This PR is based on #1740 , which should be merged first.~ => Done

### CODE REVIEW

File: integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/BridgeUsingPropertyMarkerAccessIT.java [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+/**
+ * Test that a bridge using markers to retrieve and access properties
+ * will behave as expected regarding access mode.
+ *
```
Comment (gsmet) [inline integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/BridgeUsingPropertyMarkerAccessIT.java]: A `<p>` would be in order (even if not an API, I think it's better to have a proper Javadoc).

File: util/internal/test/src/main/java/org/hibernate/search/util/impl/test/annotation/TestForIssue.java [hunk 2]
```diff
@@ -18,11 +19,18 @@
...
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ ElementType.METHOD, ElementType.TYPE })
+@Repeatable(TestForIssue.List.class)
 public @interface TestForIssue {
 	/**
 	 * The key of a JIRA issue tested.
 	 *
 	 * @return The jira issue key
 	 */
 	String jiraKey();
```
Comment (gsmet) [inline util/internal/test/src/main/java/org/hibernate/search/util/impl/test/annotation/TestForIssue.java]: In another project (see https://github.com/hibernate/hibernate-ogm/blob/master/core/src/test/java/org/hibernate/ogm/utils/TestForIssue.java#L28), we made `jiraKey` an array.

Comment (yrodiere) [reply util/internal/test/src/main/java/org/hibernate/search/util/impl/test/annotation/TestForIssue.java]: Right. I amended the commit to do the same.

File: integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/BridgeUsingPropertyMarkerAccessIT.java (hunk 1 above)
Comment (yrodiere) [reply integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/BridgeUsingPropertyMarkerAccessIT.java]: Right, done.

**Comment by yrodiere** [comment]
Merged, thanks

**Pull request merged**